api/api.py

```diff
@@ -258,11 +261,20 @@ def _format(route):
 ]
 
 
+def custom_json_serializer(obj):
+    if isinstance(obj, bson.objectid.ObjectId):
+        return str(obj)
+    elif isinstance(obj, datetime.datetime):
+        return pytz.timezone('UTC').localize(obj).isoformat()
+    elif isinstance(obj, Job):
+        return obj.map()
+    raise TypeError(repr(obj) + " is not JSON serializable")
```

Comment:
I don't understand. You are still importing util in this file. What's the problem?